charts/fluentd/templates/logger-fluentd-daemon.yaml

```diff
@@ -33,6 +33,15 @@ spec:
             memory: {{.Values.limits_memory}}
 {{- end}}
 {{- end}}
+{{- if and (.Values.syslog_host) (.Values.syslog_port)}}
```

Comment:
@rimusz we're not seeing the changes here. Are you sure you pushed your changes?